internal/pkg/utils/bililng_utils.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+)
+
+func FormatPrice(price float64, unit string) string {
+	priceStr := fmt.Sprint(price)
+
+	// Require >= 2 digits after the decimal
+	if strings.Contains(priceStr, ".") {
+		// Extend the remainder if needed
+		r := strings.Split(priceStr, ".")
+		for len(r[1]) < 2 {
```

Comment:
can we add a length check here to avoid potential panic? e.g. `invalid price format`

Reply:
Looks like go always adds decimal to floats, we have a test where the value is just integer
https://github.com/confluentinc/cli/blob/239787f52228072beffa13388feb54c5f86c28be/internal/cmd/price/command_list_test.go#L10